jinja2/tests.py

```diff
@@ -186,9 +205,21 @@ def test_in(value, seq):
     'iterable':         test_iterable,
     'callable':         test_callable,
     'sameas':           test_sameas,
-    'equalto':          test_equalto,
+    'in':               test_in,
     'escaped':          test_escaped,
-    'greaterthan':      test_greaterthan,
-    'lessthan':         test_lessthan,
-    'in':               test_in
+    '==':               test_compare_eq,
+    'eq':               test_compare_eq,
+    'equalto':          test_compare_eq,
```

Comment:
Similarly, I kept the long names for the three tests that already existed, in addition to the operator and short names.